src/PullRequestReleaseNotes/Providers/GitHubPullRequestProvider.cs

```diff
@@ -40,28 +40,28 @@ public PullRequestDto Get(string commitMessage)
             var pullRequestId = ExtractPullRequestNumber(commitMessage);
             if (pullRequestId == null)
                 return null;
-            var pullRequest = _gitHubClient.PullRequest.Get(_programArgs.GitHubOwner, _programArgs.GitHubRepository, (int) pullRequestId).Result;
-            if (pullRequest == null)
-                return null;
-            if (_programArgs.VerboseOutput)
-                Console.WriteLine($"Found #{pullRequest.Number}: {pullRequest.Title}");
-            return GetPullRequestWithLabels(pullRequest);
+            return GetPullRequestWithLabels(pullRequestId.Value);
```

Comment:
@EbenZhang is OK as long as the assumption that GitHub pull request ID will always match the GitHub Issue ID holds true. I remember checking this very early on and found that it may be possible that Issue ID may differ from Pull Request ID? I may be wrong.